Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1113,3 +1113,11 @@
     && HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)
 #define ENABLE_OPT_IN_PARTITIONED_COOKIES 1
 #endif
+
+#if !defined(ENABLE_DNS_SERVER_FOR_TESTING) && (!PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR))
+#define ENABLE_DNS_SERVER_FOR_TESTING 1
+#endif
+
+#if !defined(ENABLE_DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS) && defined(ENABLE_DNS_SERVER_FOR_TESTING)
+#define ENABLE_DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS 1
```

Comment:
Why do we need both `ENABLE_DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS` and `ENABLE_DNS_SERVER_FOR_TESTING` if they are the same platforms?

Reply:
Good catch! This has been addressed in the latest patch.

Thanks for reviewing!